src/components/tab-bar/examples/tab-bar.tsx

```diff
@@ -0,0 +1,58 @@
+import { Component, h } from '@stencil/core';
+import { Tab } from '../tab.types';
+
+const tabs: Tab[] = [
+    {
+        id: 1,
+        text: 'Joker',
+        icon: 'joker',
+        active: true,
+        iconColor: 'var(--lime-green)',
+    },
+    {
+        id: 2,
+        text: 'Parasite',
+        icon: 'insect',
+        iconColor: 'var(--lime-magenta)',
+    },
+    {
+        id: 3,
+        text: 'Harriet',
+        icon: 'administrator_female',
+        iconColor: 'var(--lime-orange)',
+    },
+    {
+        id: 4,
+        text: 'Bombshell',
+        icon: 'surprised',
+        iconColor: 'var(--lime-blue)',
+    },
+    {
+        id: 5,
+        text: 'Judy',
+        icon: 'female',
+        iconColor: 'var(--lime-deep-red)',
+    },
+    {
+        id: 6,
+        text: 'Jojo Rabbit',
+        icon: 'rabbit',
+        iconColor: 'var(--lime-yellow)',
+    },
+    {
+        id: 7,
+        text: 'Little Women',
+        icon: 'female',
+        iconColor: 'var(--lime-deep-red)',
+    },
+];
+
+@Component({
+    tag: 'limel-example-tab-bar',
+    shadow: true,
+})
+export class TabBarExample {
+    public render() {
+        return <limel-tab-bar tabs={tabs} />;
```

Comment:
I think we can improve this example a bit by using the event in some way as well. Just having a small text label below the tabs displaying the name of the current tab should be enough :slightly_smiling_face: